### ISSUE TITLE
Custom dataset with gt_masks

### ISSUE THREAD

**Issue by sedol1339** [facebookresearch/detectron2#4927]
Hello, I want to fine-tune mask R-CNN on dataset with masks. I have these masks on a disk. So, I guess I need to add a field gt_masks in my dataloader. 

However, I did not find documentation about the format of gt_masks: should it be if CHW, HWC or HW format, in float 0-1 or uint8 0-255, etc?

I guess that the only way to get this information is to create a dataloader for some of built-in datasets. However, all these datasets are large. For example, to create a dataloader for "coco_2014_minival_100" requires images in "datasets/coco/val2014" folder, so I need to wait hald a day to download 6GB val2014.zip and unpack.

Maybe i'm doing something wrong? How to solve my problem in a normal way?
